scylla/src/transport/session.rs

```diff
@@ -606,6 +606,10 @@ impl Session {
             }
         }
 
+        if response.as_schema_change().is_some() {
+            self.refresh_metadata().await?;
+        }
```

Comment:
Additionally, I'd like to see a flag added to configuration which allows to switch this feature off (it would be turned on by default) so that the users who don't like this feature can switch this off (e.g. they want to perform many schema changes in succession).